### PR TITLE
Added a reset for all user api calls on midnight.

### PR DESCRIPTION
**Pull request by namansethi**

### CODE REVIEW

**Review by Justin-JKim — APPROVED** [review]
It looks good. What does "0 0 0 * * ?" mean?

**Comment by namansethi** [comment]
> It looks good. What does "0 0 0 * * ?" mean?

This means that every day at midnight, the cron expression will triger.

File: src/test/java/SpringJPA/SchedulerTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        cal.set(Calendar.SECOND,0);
+        cal.set(Calendar.MILLISECOND,0);
+        Date correctExecutionTime = cal.getTime();
+
+        System.out.println("Next Execution date: " + df.format(nextExecutionTime));
+        System.out.println("Correct execution date: " + df.format(correctExecutionTime));
+        assertEquals(nextExecutionTime, correctExecutionTime);
+
+    }
+
```
Comment (lazarmilojevic) [inline src/test/java/SpringJPA/SchedulerTest.java:69]: Should test whether the API calls are being reset

**Review by lazarmilojevic — CHANGES_REQUESTED** [review]
add additional test(s)

File: src/test/java/SpringJPA/SchedulerTest.java (hunk 1 above)
Comment (namansethi) [reply src/test/java/SpringJPA/SchedulerTest.java:69]: Added additional test.

**Review by lazarmilojevic — APPROVED** [review]
lgtm

**Pull request merged**